test/e2e/rukpakctl_test.go

```diff
@@ -0,0 +1,297 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+	plain "github.com/operator-framework/rukpak/internal/provisioner/plain/types"
+)
+
+var _ = Describe("rukpakctl run subcommand", func() {
+	When("run executed with a valid local manifest directory", func() {
+		var (
+			ctx                  context.Context
+			bundlename           string
+			bundle               *rukpakv1alpha1.Bundle
+			bundledeploymentname string
+			bundledeployment     *rukpakv1alpha1.BundleDeployment
+		)
+		BeforeEach(func() {
+			ctx = context.Background()
+			out, err := exec.Command("sh", "-c", "../../bin/linux/rukpakctl run test ../../testdata/bundles/plain-v0/valid").Output()
```

Comment:
maybe we could pull out `../../bin/linux/rukpakctl` as a constant since its used repeatedly in the tests? also the different bundle paths could be constants as well, it would improve readability

Reply:
I'll update.  Thanks!